summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/store/DirectoryBatchedStore.scala

```diff
@@ -81,7 +81,7 @@ class DirectoryBatchedStore[K <: Writable, V <: Writable](val rootPath: String)(
 
         val lastBatchStatus =
           hdfsPaths.map(getFileStatus(_, conf))
-            .filter { input => input._1 && (BatchID(input._2) < exclusiveUB) }
+            .filter { input => input._1 && (BatchID(input._2.milliSinceEpoch) < exclusiveUB) }
```

Comment:
Thats what I thought too, but actually this is what we were doing here before too. 

BatchID's constructor that I removed as part of this (had to for AnyVal erasure reasons):
 def apply(ts: Timestamp) = new BatchID(ts.milliSinceEpoch)

So this prior code here was actually doing semantically the same as this new code